### DISCUSSION TITLE
Dark mode/theme

### DISCUSSION THREAD

**Discussion by ajpixs** [lllyasviel/Fooocus#136] [Q&A]
my eyes are bloodshot from the white screen, is there any possible way to set a dark mode/theme?

update: answered my own question, nm lol.

**Comment by greatwolf** [comment]
What was the answer? Passing `--theme dark` to launch.py doesn't work which is usually how it's done in A1111.

**Reply by Snowdog37** [reply]
https://github.com/lllyasviel/Fooocus/discussions/952

--theme=dark

**Reply by ppricha75** [reply]
it's not working, error message when launching sad :-((

**Reply by mikejarxyz** [reply]
me also plz dark mooode

**Comment by StarPet** [comment]
Add "?__theme=dark" to your URL
